scripts/utils.js

```diff
@@ -79,6 +80,15 @@ function printLog(...params) {
   console.log(greenColor, ...params, resetColor)
 }
 
+async function http(url) {
```

Comment:
what about naming it `fetchWrapper` and exposing it as `fetch`?

Reply:
Hum, why not